application/helpers/remotecontrol/remotecontrol_handle.php

```diff
@@ -2699,6 +2699,7 @@ public function cpd_importParticipants($sSessionKey, $aParticipants)
                 'lastname' => $aParticipantData['lastname'],
                 'email' => $aParticipantData['email'],
                 'owner_uid' => Yii::app()->session['loginID'], // ToDo is this working?
+                'language' => $aParticipantData['language'], 
             );
```

Comment:
Should participant_id be a field that is allowable so that added Survey {participant_id}s map to the cpd?